parser/src/java/org/apache/hadoop/hive/ql/parse/HiveLexerParent.g

```diff
@@ -397,6 +397,7 @@ KW_SNAPSHOTS: 'SNAPSHOTS';
 KW_RETAIN: 'RETAIN';
 KW_RETENTION: 'RETENTION';
 KW_TAG: 'TAG';
+KW_FASTFORWARD: 'FASTFORWARD';
```

Comment:
could we rename to 'FAST-FORWARD', at least that is a term used in git

Reply:
or maybe call it simple 'MERGE', see https://blog.mergify.com/what-is-a-git-merge-fast-forward/